wbuild/createIndex.py

```diff
@@ -206,4 +204,8 @@ def ci():
 
     if os.path.exists(libDir):
         shutil.rmtree(libDir)
-    shutil.copytree('.wBuild/lib', libDir)
+
+
+    wbuildPath = pathlib.Path(wbuild.__file__).parent
+    shutil.copytree(str(wbuildPath) + "/html/lib", libDir) ### quick fix for line below
```

Comment:
should be just htmlOutputPath + '/lib' instead of wbuildPath + "/html/lib"

Reply:
I don't think that's correct, since we want to copy from `wbuildPath/..` to `libDir` (= `htmlOutputPath + "/lib"`)